frontend/src/components/common/authed-menu/authed-menu.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'common/types/types';
+import { getValidClasses } from 'helpers/helpers';
+import { useLocation } from 'hooks/hooks';
+import { Link } from 'components/common/common';
+import { routes } from './common';
+import styles from './styles.module.scss';
+
+const AuthedMenu: FC = () => {
+  const { pathname } = useLocation();
+
+  return (
+    <div className={styles.menu}>
+      {routes.map(({ name, subroutes }, idx) => (
+        <div key={name}>
+          <h4 className={styles.title}>{name}</h4>
+          <div
+            className={getValidClasses(
+              styles.links,
+              idx !== routes.length - 1 && styles.bottomLine,
```

Comment:
I think you could achieve this effect by css-selectors (`:not(:last-child)` or something like that)
```suggestion

```